src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Resources/Base/BuildReleaseArm64XFormsDotNet.apkdesc

```diff
@@ -2477,5 +2477,5 @@
       "Size": 812848
     }
   },
-  "PackageSize": 10864931
+  "PackageSize": 10897699
```

Comment:
There is a small app size change here.

It looks like some are:
* Past .NET bumps were under the threshold
* `liblibmonodroid.so` changed, which must be other recent native code changes
* We are actually preserving more stuff now. Perhaps we had som subtle trimmer bugs?

Reply:
@jonathanpeppers [wrote](https://github.com/xamarin/xamarin-android/pull/8758/files#r1503060254):

> We are actually preserving more stuff now. Perhaps we had some subtle trimmer bugs?

*Are* we preserving more stuff?  What stuff and why?  I think this might warrant more investigation.  Consider [my earlier comment](https://github.com/xamarin/xamarin-android/pull/8758/files#r1507994791): we should *not* be preserving all public methods on the `Resource` type!  That also is apparently the case: we're *not* preserving all public methods, as indicated by the fact that `UnnamedProject.dll` is *smaller*, as is `_Microsoft.Android.Resource.Designer.dll` (by two bytes!).

`Java.Interop.dll` increased in size by ~3KB, while `Mono.Android.dll` *shrank* by ~2KB.

Of interest is that `classes*.dex` also increased by ~7KB:

```diff
    "classes.dex": {
-     "Size": 9458972
+     "Size": 9418292
    },
    "classes2.dex": {
-     "Size": 103648
+     "Size": 150904
    },
```

This implies that we're emitting Java Callable Wrappers for more types.  What types are we now emitting JCWs for, and why?  *That* could be the "subtle trimmer bug" you elude to, and I would like more details here.